src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -945,4 +945,26 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_ColumnSortedDescendingHelpText" xml:space="preserve">
     <value>Sorted descending, press space to sort ascending</value>
   </data>
+  <data name="Label_PackageDeprecatedToolTip" xml:space="preserve">
+    <value>This package version is deprecated.</value>
+  </data>
+  <data name="Label_PackageSelectForInfoToolTip" xml:space="preserve">
+    <value>Select to view package details for more information.</value>
+  </data>
+  <data name="Label_PackageVulnerableToolTip" xml:space="preserve">
```

Comment:
Since the placeholder will contain a localized string, either:

- Add a thrid resource string "{0}{1}", where 0 will be the message and {1} will be the severy. That way, loc team can reorder the sentence as needed
- Create one string resource for each Severity Level.

See: https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/475/String-Concatenation

Reply:
Thanks Fernando! It looks like I'll need a string resource for each severity level with the severity embedded since it's in the middle of the string. Does that sound right to you?

Reply:
Upon further consideration, I realize that this string does not actually involve string concatenation and falls within this rule:
```
One string is embedded as a token in the other and when commented appropriately, the translation vendor has context on the entire string and the ability to reorder the parts and make other changes as needed:

Resource: "String1a {0}string1b"
Comment: '{0}' represents X and can be replaced by values {a, b & c}
```

By creating string resources with a placeholder and describing in the comment what the placeholder stands for, the translator is at liberty to rearrange the sentence in another language so that the severity text fits within the sentence.

Reply:
Upon further consideration, I realize that this string does not actually involve string concatenation and falls within the rule regarding embedded strings. By creating string resources with a placeholder and describing in the comment what the placeholder stands for, the translator is at liberty to rearrange the sentence in another language so that the severity text fits within the sentence.